keps/2724-topology-aware-scheduling/README.md

```diff
@@ -1168,3 +1167,32 @@ type WorkloadStatus struct {
 Uncertanity about the final shape of the feature and the required format.
 We will decide on the format of this field based on the feedback from the 
 customers on the MVP.
+
+### MostFreeCapacity algorithm
```

Comment:
@lchrzaszcz Could you move `| TASProfileMostFreeCapacity (deprecated)  | MostFreeCapacity  | MostFreeCapacity  | MostFreeCapacity  |` table to here?